packages/ERTP/src/issuer.js

```diff
@@ -32,6 +30,12 @@ function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT) {
         });
       },
       getAllegedName: () => allegedName,
+
+      // Give information to UI on how to display the amount.
+      // Fungible digital assets should be represented in integers, in
+      // the smallest unit (i.e. USD might be represented in mill,
+      // a thousandth of a dollar. In that case, `decimals` would be 3).
+      decimals: () => decimals,
```

Comment:
Our general convention is that getter *methods*, i.e., those that are explicitly called with parens, are `getFoo`. A property named `foo` should be expected to be used as `obj.foo` to obtain the foo value, whether it is a data property or an accessor property. A JavaScript example that trips be up every time: The size of a map is `map.size` rather than `map.size()`.

```suggestion
      getDecimals: () => decimals,
```
combined with my other suggestion
```js
  getOptions: () => options,
``` 
or maybe a name more specific than `options` but still not specific to `nat`.

Reply:
How about 'presentationHints' or something in that direction?

Reply:
how about `getDisplayData`?